axolotl-web/src/components/VueOtpBox.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div id="wrapper" :style="{ justifyContent: direction }">
+    <div v-for="(char, index) in arraySize" v-bind:key="'otp_' + index">
+      <input
+        class="inputBox"
+        type="tel"
+        :key="index"
+        maxlength="1"
+        v-model="arraySize[index]"
+        @keydown="
+          handleEnterKey($event);
+          handleKeyDown($event, index);
+        "
+        @input="handleInput($event, index)"
+        @paste="onPaste"
+        :autofocus="index === 0"
+        :ref="`otp${index}`"
+        @focus="newColor(index)"
+        @blur="defaultColor(index)"
+      />
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: "VueOtp",
```

Comment:
Is Otp an abbreviation of something? (I can think of One Time Pad). If it is an abbreviation, what about using uppercase for the naming? Additionally, what extra value does the `Vue` bring in the component name? Considering that all components are written with Vue, I see using it as a prefix superfluous.

Reply:
Otp means One time password and the component is mostly copied from a vue2 module, that's the reason, because non the existing npm modules are ready for vue3